pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -364,6 +365,35 @@ func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems
 	return helper.ToUnstructured(pod)
 }
 
+func aggregatePersistentVolumeStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pv := &corev1.PersistentVolume{}
+	err := helper.ConvertToTypedObject(object, pv)
+	if err != nil {
+		return nil, err
+	}
+	newStatus := &corev1.PersistentVolumeStatus{}
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+		temp := &corev1.PersistentVolumeStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+		klog.V(3).Infof("Grab persistentVolume(%s/%s) status from cluster(%s), phase: %s",
+			pv.Namespace, pv.Name, item.ClusterName, temp.Phase)
+		newStatus.Phase = temp.Phase
```

Comment:
The `newStatus.Phase` will be overwritten over loops, and only keeps the last `temp.phase`. I don't think this is we expected behavior. 

Maybe you can check how we deal with this similar thing at `aggregatePodStatus()`.

Reply:
Sure, let me push another commit

Reply:
test added